dotcom-rendering/src/web/components/NavList.tsx

```diff
@@ -0,0 +1,74 @@
+import { css } from '@emotion/react';
+import {
+	between,
+	body,
+	border,
+	from,
+	palette,
+	space,
+	until,
+} from '@guardian/source-foundations';
+import { Link } from '@guardian/source-react-components';
+import type { DCRContainerPalette } from 'src/types/front';
+import type { TrailType } from '../../types/trails';
+import { decideContainerOverrides } from '../lib/decideContainerOverrides';
+
+type Props = {
+	trails: TrailType[];
+	containerPalette?: DCRContainerPalette;
+};
+
+const ulStyles = css`
+	${until.tablet} {
+		column-count: 1;
```

Comment:
Very neat!

Reply:
Nice one! For info, [`column-count`](https://caniuse.com/mdn-css_properties_column-count) is well supported, but there are quite a few caveats around the [CSS3 multicolumn implementations](https://caniuse.com/multicolumn).